src/commands/install.yml

```diff
@@ -6,11 +6,6 @@ parameters:
     type: enum
     enum: ["1", "2"]
     default: "2"
-  skip-install-check:
```

Comment:
I was thinking, could this still be useful in situations when e.g. aws v2 is already installed on an image, but it's an older version that you'd like to overwrite with the latest?

Reply:
Ya we can do that, If I had a better method of matching regex in a way that worked on macos as well I think we could still automate that. We can just add it back for now, if selected, we'll set the internal version to zero, so that it will always be installed.

Reply:
re-added